recipe/build.sh

```diff
@@ -30,6 +31,19 @@ cp -R $RECIPE_DIR/tf_third_party/* $SRC_DIR/third_party/
 # this must use commas to separate libs, otherwise bazel breaks inscrutably
 export TF_SYSTEM_LIBS="com_google_absl,zlib"
 
+# Hacky workaround to fix some dependency issues with bazel
+for f in dist/BUILD.bazel dist/dist.bzl; do
+  sed -i '/@system_python\/\/:version\.bzl/d' $f
+  sed -i "s|SYSTEM_PYTHON_VERSION|\"${PY_VER//./}\"|g" $f
+done
+# https://github.com/protocolbuffers/protobuf/issues/22313
+sed -i "s|SUPPORTED_PYTHON_VERSIONS\[-1\]|\"${PY_VER}\"|g" ../MODULE.bazel
+sed -i '/SUPPORTED_PYTHON_VERSIONS *= *\[/,/]/ s/^\( *\]\)/    "3.13",\n\1/' ../MODULE.bazel
+if [[ "${target_platform}" == osx-* ]]; then
+  # Upgrade to a newer rules_python
+  sed -i 's/\(bazel_dep(name *= *"rules_python", *version *= *"\)[^"]*\(")\)/\11.5.0\2/' ../MODULE.bazel
+fi
```

Comment:
OK, so we're adding 3.13 to the list [here](https://github.com/protocolbuffers/protobuf/blob/v31.1/MODULE.bazel#L112-L117) (why isn't it there already...?! 🤔), and ensuring it's [considered](https://github.com/protocolbuffers/protobuf/blob/v31.1/MODULE.bazel#L127-L135) the default